automatic/ruby.portable/update.ps1

```diff
@@ -0,0 +1,65 @@
+import-module au
+
+$releases = 'https://rubyinstaller.org/downloads/archives/'
+
+function global:au_SearchReplace {
+   @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(?i)(^\s*packageName\s*=\s*)('.*')"  = "`$1'$($Latest.PackageName)'"
```

Comment:
Don't replace the package name, instead use the requested change for the install script.